demo.sh

```
@@ -1,82 +1,82 @@
-fi
-
-echo "========================================="
-echo "Demo Complete!"
-echo "========================================="
-echo ""
-echo "To run NexumDB interactively:"
-echo "  export PYO3_USE_ABI3_FORWARD_COMPATIBILITY=1"
-echo "  ./target/release/nexum"
-echo ""
-echo "For more information, see README.md and walkthrough.md"
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)